GUI/MainWindow.xaml

```diff
@@ -224,7 +224,18 @@
                         Background="Silver"
                         Height="5" />
                 <GroupBox Header="Notifications" Grid.Row="2">
-                    <RichTextBox x:Name="notificationsTextBox" VerticalScrollBarVisibility="Visible" IsReadOnly="True">
+                    <RichTextBox x:Name="notificationsTextBox" VerticalScrollBarVisibility="Visible" IsReadOnly="True" IsDocumentEnabled="True" >
```

Comment:
can we add some kind of check that writes this only the first time someone opens MetaMorpheus?

Reply:
would clearing the notifications textbox when "run all tasks" is clicked work?

Reply:
that would still output this every time MetaMorpheus is run - I'd prefer if this was only output the first time it was run. Try adding a `bool` in `GuiGlobalParams`. Maybe like `bool FirstTimeRunning = true;` then when MM is opened, set it to `false`. I think it will reset every time MetaMorpheus is updated but that's fine

Reply:
how would the bool value translate to the GUI? i read something about data triggers but i'm not entirely sure.